waspc/src/Wasp/Generator/WebAppGenerator/Auth/LocalAuthG.hs

```diff
@@ -10,14 +10,10 @@ import Wasp.Generator.AuthProviders (localAuthProvider)
 import Wasp.Generator.AuthProviders.Local (serverLoginUrl, serverSignupUrl)
 import Wasp.Generator.FileDraft (FileDraft)
 import Wasp.Generator.Monad (Generator)
-import Wasp.Generator.WebAppGenerator.Auth.Common (getOnAuthSucceededRedirectToOrDefault)
 import Wasp.Generator.WebAppGenerator.Common as C
-import Wasp.Util ((<++>))
 
 genLocalAuth :: AS.Auth.Auth -> Generator [FileDraft]
-genLocalAuth auth =
-  genActions auth
-    <++> genForms auth
+genLocalAuth = genActions 
```

Comment:
Did we delete the files with forms as well?

Reply:
I actually moved form generation logic to `WebAppGenerator/AuthG.hs` since form now contains both local and external stuff. So we're still using the files.